Makefile

```diff
@@ -307,6 +307,8 @@ go-generate:
 test tests:
 	@echo "Run ginkgo"
 	ginkgo run -r $(ginkgo_flags)
+	@echo "Run controller unit test"
+	go test -v ./internal/controllers/...
```

Comment:
As I explain in other comments I'd suggest to use the Ginkgo framework for these tests, that way we don't have to different ways to run tests and display the results, and these two lines wouldn't be needed.